src/lib/utils.test.ts

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The Pepr Authors
+
+import { expect, test, describe } from "@jest/globals";
+import { convertToBase64Map, convertFromBase64Map, base64Decode, base64Encode } from "./utils";
+
+describe("utils", () => {
+  test("convertToBase64Map should encode all ascii values and skip listed values in skip array", () => {
+    const obj = {
+      data: {
+        test1: "test1",
+        test2: "test2",
+        test3: "test3",
+        test4: "test4",
+      },
+    };
+    const skip = ["test2", "test4"];
+    convertToBase64Map(obj, skip);
+    expect(obj.data["test1"]).toBe(base64Encode("test1"));
+    expect(obj.data["test2"]).toBe("test2");
+    expect(obj.data["test3"]).toBe(base64Encode("test3"));
+    expect(obj.data["test4"]).toBe("test4");
+  });
+
```

Comment:
this improves the branchiness:

+  test("convertToBase64Map empty object", () => {
+    const obj = {};
+    const objOut = {data:{}};
+    convertToBase64Map(obj, []);
+    expect(obj).toStrictEqual(objOut);
+  });
+